### ISSUE TITLE
Handle retries better

### ISSUE THREAD

**Issue by RamVasuthevan** [RamVasuthevan/whois-server-list#182]
See more:
https://github.com/RamVasuthevan/whois-server-list/blob/main/whois-server-list-generator/generate_whois_servers.py#L129
https://github.com/RamVasuthevan/whois-server-list/actions/runs/7454835419

**Comment by RamVasuthevan** [comment]
We are having failed runs which look like this:
```
Error occurred while fetching coupon. Retrying...
coupons
courses
cpa
cr
credit
creditcard
creditunion
cricket
crown
crs
cruise
Error occurred while fetching cruise. Retrying...
Error occurred while fetching cruise. Retrying...
Error occurred while fetching cruise. Retrying...
Error occurred while fetching cruise. Status Code: 500
```

**Comment by RamVasuthevan** [comment]
I think I fixed this #183
